packages/spruce-skill-server/gql/listeners.js

```diff
@@ -26,7 +27,7 @@ module.exports = (koa, gqlOptions) => {
 
 	listenerPaths.forEach(path => {
 		try {
-			log.info(`Importing gql listener: ${path}`)
+			debug(`Importing gql listener: ${path}`)
```

Comment:
It won't print out every file it imports by default and will only report errors.

Unless `DEBUG=spruce-skill-server` is set, then it will be verbose.  This is the same pattern we follow elsewhere for verbose logs in packages